lib/spack/spack/build_environment.py

```diff
@@ -109,7 +109,7 @@ def __call__(self, *args, **kwargs):
         disable = env_flag(SPACK_NO_PARALLEL_MAKE)
         parallel = not disable and kwargs.get('parallel', self.jobs > 1)
 
-        if parallel:
+        if parallel and 'bazel' not in self.command:
```

Comment:
> What if we renamed it to ParallelExecutable()?

Consider this `-pedantic` mode, but I'll avoid that in case we ever need to wrap an MPI executable. What about `ParallelBuildCommand` (or something along that line) ?